src/libs/EmojiUtils.ts

```diff
@@ -424,7 +425,7 @@ function suggestEmojis(text: string, lang: Locale, limit: number = CONST.AUTO_CO
     for (const node of nodes) {
         if (node.metaData?.code && !matching.find((obj) => obj.name === node.name)) {
             if (matching.length === limit) {
-                return matching;
+                return lodashSortBy(matching, (emoji) => !emoji.name.includes(emojiData[0].toLowerCase().slice(1)));
```

Comment:
Could you please separate the sorting function to avoid duplicating logic? ST like this
```diff
+   const sortByName = (emoji: Emoji) => !emoji.name.includes(emojiData[0].toLowerCase().slice(1));
    for (const node of nodes) {
        if (node.metaData?.code && !matching.find((obj) => obj.name === node.name)) {
```
```suggestion
                return lodashSortBy(matching, sortByName);
```

Reply:
@suneox i updated, please check again